ush/parsing_model_configure_FV3.sh

```diff
@@ -25,6 +25,7 @@ local SHOUR=${model_start_date:8:2}
 local FHROT=${IAU_FHROT:-0}
 local DT_ATMOS=${DELTIM}
 local RESTART_INTERVAL="${FV3_RESTART_FH[*]}"
+local RESTART_FH="${CMEPS_RESTART_FH:-" "}"
```

Comment:
I am not sure introducing a `CMEPS` variable dependency in `FV3` would be clean, since that means there is a dependency on a coupled model variable for an atm-only configuration.  Need to think a bit on this.

Reply:
The RESART_FH variable is not used by FV3, however it is in the model_configure file.

Reply:
Also - please note this is what is being done in this PR: https://github.com/ufs-community/ufs-weather-model/pull/2419 so if you want changes we'll potentially need changes to ufs-weather-model

Reply:
If the FV3 does not use this variable, can we just set this as:
`local RESTART_FH=" "`

Reply:
This assumes the restarts are written according to `local RESTART_INTERVAL="${FV3_RESTART_FH[*]}"`, correct?

Reply:
The RESTART_FH variable is needed for the non-FV3 components to write restarts at the FV3 restart times when we have IAU, so no we cannot have it as RESTART_FH=" " to my understanding.

Reply:
I misunderstood [this](https://github.com/NOAA-EMC/global-workflow/pull/3190/files#r1900186913) comment that said the FV3 does not use this variable. Does the `model_configure` also now use this variable to specify restart write times for all model components? (In the past, `model_configure` was fv3atm specific)
How does this relate to `RESTART_INTERVAL` that is currently specifying the FV3 restart times? Does `RESTART_FH` override `RESTART_INTERVAL`?

Reply:
From @NickSzapiro-NOAA in an email: 

```
3 restart triggers:
restart_n+restart_option: frequency-based ESMF restart trigger (via ufs.configure) *Not currently in FV3
restart_interval: just in FV3 (via model_configure) and can mean different things based on "encoding"
restart_fh: UFS wraps ESMF trigger (via model_configure). *Not currently in FV3
```

Reply:
@aerorahul - let me know if you'd like me to run a test case and save the output showing that we're writing all restarts at the same time when using IAU for all components.  I can also run an fv3 standalone ci test to ensure that we are not negatively impacting atm-only runs too.